scipy/optimize/_linprog_ip.py

```diff
@@ -87,13 +88,15 @@ def _get_solver(M, sparse=False, lstsq=False, sym_pos=True,
                 def solve(r, sym_pos=False):
                     return sps.linalg.lsqr(M, r)[0]
             elif cholesky:
-                try:
-                    # Will raise an exception in the first call,
-                    # or when the matrix changes due to a new problem
-                    _get_solver.cholmod_factor.cholesky_inplace(M)
-                except Exception:
-                    _get_solver.cholmod_factor = cholmod_analyze(M)
-                    _get_solver.cholmod_factor.cholesky_inplace(M)
+                with np.testing.suppress_warnings() as sup:
```

Comment:
Would you be able to add the same to-do comment here please. Also I think it would be helpful to into a reference to your pr so it's easier for someone to check on progress

Reply:
I added a TODO comment (with link to this PR and reverted code present, but commented out).
Added link to the PR in the other TODO comment as well.